### DISCUSSION TITLE
【汇总记录】关于不同学院的不同要求

### DISCUSSION THREAD

**Discussion by xkwxdyy** [whutug/whu-thesis#218] [Ideas]
受 https://github.com/whutug/whu-thesis/issues/217#issuecomment-1565421824 启发，开此 discussion 用于记录一些不同学院对于模版细节的不用要求，用于更好地开发模版。

**Comment by entr0pia** [comment]
网安学院要求
1. 硕士论文的引用标识为上标数字
2. 英文标题页如图, 除题目外, 应该是四号字. 但是我看了我们实验室往年的毕业论文, 也不全是这样
![image](https://github.com/whutug/whu-thesis/assets/30486766/2b523a05-5f00-4871-9264-e8c6385432bb)

default: 印制规定中的未定义行为, 以Word默认格式为准 (这条是猜的)

**Reply by note286** [reply]
请给出学校/学院规范链接，如果没有公开，例如在班级群发布，请直接提供Microsoft Word模板文件或者规范文件。

**Reply by entr0pia** [reply]
没有模板, 没有文件, 只有老师的一张嘴...

**Reply by note286** [reply]
如果没有相关规范，只有某个老师或者某几个老师的口述，如何确保这是统一的认识？社区是没法开展开发工作的。

**Reply by taoxiom** [reply]
> 如果没有相关规范，只有某个老师或者某几个老师的口述，如何确保这是统一的认识？社区是没法开展开发工作的。

确实，只凭口述那也只是个别老师的意见，模板本来就不可能做到面面俱到的。

**Comment by H0radricCube** [comment]
[武汉大学计算机学院研究生学位论文写作要求和规范](https://cs.whu.edu.cn/info/1140/4335.htm)

这个是 23 年计算机学院硕士毕业论文采用的要求，确实在英文封面、页眉横线等位置有所不同。希望别挂我 QAQ

**Comment by wangzhankun** [comment]
计算机学院研究生学位论文写作要求与规范，网安学院与此相同
[计算机学院研究生学位论文写作要求与规范（2023版）.zip](https://github.com/whutug/whu-thesis/files/14134582/default.zip)
